airbyte-integrations/connectors/source-google-search-console/BOOTSTRAP.md

```diff
@@ -0,0 +1,55 @@
+# Google Search Console
+
+The API docs: https://developers.google.com/webmaster-tools/search-console-api-original/v3/parameters.
+
+## Endpoints and Streams:
+
+1. [Site](https://developers.google.com/webmaster-tools/search-console-api-original/v3/sites) – Full refresh
+2. [Sitemaps](https://developers.google.com/webmaster-tools/search-console-api-original/v3/sitemaps) – Full refresh
+3. [Analytics](https://developers.google.com/webmaster-tools/search-console-api-original/v3/searchanalytics) – Full refresh, Incremental
+
+There are multiple streams in the `Analytics` endpoint. 
+We have them because if we want to get all the data from the GSC (using the SearchAnalyticsAllFields stream), 
+we have to deal with a large dataset. 
+
+In order to reduce the amount of data, and to retrieve a specific dataset (for example, to get country specific data) 
+we can use SearchAnalyticsByCountry.
+
+There are:
+   1. SearchAnalyticsByDate
+   2. SearchAnalyticsByCountry
+   3. SearchAnalyticsByPage
+   4. SearchAnalyticsByQuery
+   5. SearchAnalyticsAllFields
+
+## Authorization
+
+There are 2 types of authorization `User Account` and `Service Account`.
+To chose one we use an authorization field with the `oneOf` parameter  in the `spec.json` file.
+
+
+## Analytics
```

Comment:
I think this section should be in the source code since if the code changes this can go out of date, so putting it next to the code reduces the chance it goes out of date. . The bootstrap should be more focused on higher level details of how the API works